src/navigator/types.tsx

```diff
@@ -26,8 +26,8 @@ import { WalletConnectRequestType } from 'src/walletConnect/types'
 // Typed nested navigator params
 type NestedNavigatorParams<ParamList> = {
   [K in keyof ParamList]: undefined extends ParamList[K]
-    ? { screen: K; params?: ParamList[K] }
-    : { screen: K; params: ParamList[K] }
+    ? { tab: K; params?: ParamList[K] }
+    : { tab: K; params: ParamList[K] }
```

Comment:
`screen` is really a proper noun in the wallet and these are not screens these are tabs. Using this opportunity to fix this misname

Reply:
I agree that these look more like tabs, but whenever I switch between the two, a screen change redux action is fired, so these should still be screens? Otherwise there will be a type mismatch here:

https://github.com/valora-inc/wallet/blob/765c5594d69e906ca054b9f5237e548ee4c8ee65/src/app/reducers.ts#L23

```
 DEBUG  [20:49:29.170Z] iPhone 13 redux/saga@logger {"activeScreen": "QRScanner", "type": "APP/ACTIVE_SCREEN_CHANGED"}
 DEBUG  [20:49:30.052Z] iPhone 13 ValoraAnalytics Analytics is disabled, not tracking screen QRCode
 DEBUG  [20:49:30.063Z] iPhone 13 redux/saga@logger {"activeScreen": "QRCode", "type": "APP/ACTIVE_SCREEN_CHANGED"}
```

Reply:
ah yea missed that. I'll keep the prop name for QRNavigator as `tab` but change the type back to Screens